DQMOffline/MuonDPG/plugins/GEMTnPEfficiencyTask.cc

```diff
@@ -0,0 +1,848 @@
+/*
+ * \file GEMTnPEfficiencyTask.cc
+ * \author Qianying
+ *
+ * \interited from the TnP framework of  
+ * \author L. Lunerti - INFN Bologna
+ *
+ */
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/MuonReco/interface/MuonSegmentMatch.h"
+#include "DataFormats/MuonReco/interface/MuonGEMHitMatch.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
```

Comment:
Is this used?

Reply:
Hi Perrotta, the deltaPhi is not used and I removed it. MuonGEMHitMatch is used to calculate the dx between the propagated track and the hit. and the MuonSegmentMatch is used for the calculation of the propagated track and the segment. So I keep the two.